doc/instrumentation.rst

```diff
@@ -76,39 +76,39 @@ Here ``<names>`` is a comma-separated list of instrumentation backends. For exam
 
    $ dune build --instrument-with bisect_ppx,landmarks
 
-This will instruct Dune to activate the given backend globally, i.e. in all
+This will instruct Dune to activate the given backend globally, i.e., in all
 defined build contexts.
 
-It is also possible to enable instrumentation backends via the
-``dune-workspace`` file, either globally, or for specific builds contexts.
+It's also possible to enable instrumentation backends via the
+``dune-workspace`` file, either globally or for specific builds contexts.
 
-To enable an instrumentation backend globally, you can type in your
+To enable an instrumentation backend globally, type the following in your
 ``dune-workspace`` file:
 
 .. code:: scheme
 
-   (lang dune 3.0)
+   (lang Dune 3.0)
```

Comment:
should be reverted. This indented block is the literal contents of a dune-workspace file. It has its own conventions and in this case it requires dune to be lowercased.

Reply:
Great! Thanks for pointing this out.